### PR TITLE
WSTEAMA-1261-width parameter appearing in Billboard URL's means image being rendered is the fallback

### PR DESCRIPTION
**Pull request by Nabeel1276**
Resolves (https://jira.dev.bbc.co.uk/browse/WSTEAMA-1261)

Overall changes
======
Hard coded 240 as the value for the image width and updated fixtures to ensure tests pass

Code changes
======
Used replace function to replace the width with a hard coded string of 240

Testing
======
Ensure when inspecting a page with a Billboard or messageBanner (i.e Hindi), the width value on the image url is 240 (px)
Open up the Developer Tools in your browser, then using the web developer extension disable all CSS styles and check the size of the Billboard image. It should no longer be huge.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

**Comment by Nabeel1276** [comment]
This bug is also present in the MessageBanner component, so it worth changing that too? (given what I have done is correct)

P.S. This has been done!

**Review by shayneahchoon — APPROVED** [review]

File: src/app/components/MessageBanner/index.test.tsx [hunk 1]
```diff
@@ -104,7 +104,7 @@ describe('MessageBanner', () => {
       );
       const image = screen.getByAltText('');
       expect(image.getAttribute('src')).toEqual(
-        summary.imageUrl.replace('{width}', 'raw'),
+        summary.imageUrl.replace('{width}', '240'),
```
Comment (HarveyPeachey) [inline src/app/components/MessageBanner/index.test.tsx]: You should hard code this value, it's not great practice to use the implementation logic as part of the test

Comment (HarveyPeachey) [reply src/app/components/MessageBanner/index.test.tsx]: Actually thinking about this, is there a reason this isn't being done in the BFF?

**Review by HarveyPeachey — APPROVED** [review]
Nice

File: src/app/components/MessageBanner/index.test.tsx (hunk 1 above)
Comment (amoore108) [reply src/app/components/MessageBanner/index.test.tsx]: I think these all come back from the BFF as a list of Curations, so the same data is used to display a normal promo as well as the Billboard/MessageBanner components. Could maybe have a lookup table in the BFF that has the resolution to use based on the curation-type, but I'm not too familiar with how its all setup.

**Pull request merged**